source/crud/tests/unified/aggregate-out-readConcern.yml

```diff
@@ -0,0 +1,200 @@
+# This file was created automatically using mongodb-spec-converter.
+# Please review the generated file, then remove this notice.
+
+description: aggregate-out-readConcern
+schemaVersion: '1.1'
+createEntities:
+  -
+    client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      uriOptions:
+        retryReads: false
+      observeEvents:
+        - commandStartedEvent
+  -
+    database:
+      id: &database0 database0
+      client: client0
+      databaseName: &database_name crud-v2
+  -
+    collection:
+      id: &collection0 collection0
+      database: database0
+      collectionName: &collection_name test_aggregate_out_readconcern
+  -
+    collection:
+      id: &collection_readConcern_majority collection_readConcern_majority
+      database: database0
+      collectionName: *collection_name
+      collectionOptions:
+        readConcern: { level: "majority" }
+  -
+    collection:
+      id: &collection_readConcern_local collection_readConcern_local
+      database: database0
+      collectionName: *collection_name
+      collectionOptions:
+        readConcern: { level: "local" }
+  -
+    collection:
+      id: &collection_readConcern_available collection_readConcern_available
+      database: database0
+      collectionName: *collection_name
+      collectionOptions:
+        readConcern: { level: "available" }
+  -
+    collection:
+      id: &collection_readConcern_linearizable collection_readConcern_linearizable
+      database: database0
+      collectionName: *collection_name
+      collectionOptions:
+        readConcern: { level: "linearizable" }
+  -
+    collection:
+      id: &collection_readConcern_invalid collection_readConcern_invalid
+      database: database0
+      collectionName: *collection_name
+      collectionOptions:
+        readConcern: { level: "!invalid123" }
```

Comment:
This is problematic for any driver than uses an enum for ReadConcern level, as there is no way to create a ReadConcern with an invalid level.  Because of that, the test runner will fail to create this entity, and every test in this file will fails as a result.

If it's important to test invalid read concern levels for drivers that allow them, I suggest moving the test that uses this entity (and the entity itself) to a separate file, so that drivers can skip the test.

I'm also fine with not testing this case.

Reply:
How were you previously passing the [v2 test](https://github.com/mongodb/specifications/blob/master/source/crud/tests/v2/aggregate-out-readConcern.yml#L95)? I assume `error: true` was satisfied by an error attempting to construct the collection object for the operation, without ever getting around to invoking the `aggregate` method?

I don't have any objections to removing this test, since it seems like something that belongs in the [read/write concern spec tests](https://github.com/mongodb/specifications/tree/master/source/read-write-concern/tests) if anything.

Reply:
Grepping, I found this:

                        skipTest(testDocument, test.asDocument())
                                // Skip test because is assumes the driver sends an invalid read concern level to the server.  But the
                                // Java driver uses an enum for read concern level so it's a client-side not a server-side error
                                || description.equals("invalid readConcern with out stage")});

Reply:
I can't do the same thing here because the invalid read concern is in the entities section, which applies to all the tests in the file not just the one that uses that entity

Reply:
I can remove that test.